verilog/analysis/checkers/signal_name_style_rule.cc

```diff
@@ -43,17 +48,37 @@ using verible::LintViolation;
 using verible::SyntaxTreeContext;
 using verible::matcher::Matcher;
 
-static constexpr absl::string_view kMessage =
-    "Signal names must use lower_snake_case naming convention.";
+#define STYLE_DEFAULT_REGEX "[a-z_0-9]+"
+static std::string style_default_regex = STYLE_DEFAULT_REGEX;
```

Comment:
Please don't use global `std::string` as it allocation and deallocation these is a bit icky. A `static constexpr absl::string_view` would be fine.
(also, assign that directly, no need for the `#define`. This should work below in the `.param` init).

Reply:
Done